src/qt/rpcconsole.h

```diff
@@ -70,6 +70,8 @@ class RPCConsole: public QWidget
 
     QString tabTitle(TabTypes tab_type) const;
     QKeySequence tabShortcut(TabTypes tab_type) const;
+    const QString tr_yes{tr("Yes")}, tr_no{tr("No")}, tr_to{tr("To")}, tr_from{tr("From")},
+        tr_ban_for{tr("Ban for")}, tr_na{tr("N/A")}, tr_never{tr("Never")}, tr_unknown{tr("Unknown")};
```

Comment:
Maybe introduce a namespace instead of `tr_` prefix?

Reply:
Sure. Would you provide a working example of what you're thinking of?

Reply:
https://github.com/hebasto/gui/commit/9c7668050b872cbf2c755e3f945cc2a07d763b44

Reply:
If that idea is not that you like, could you, at least, make these variables `private`?

Reply:
Thanks! I would not have guessed that version, so your branch saved time. LGTM, done, added co-credit.